.github/workflows/wheel_win_x86_64.yml

```diff
@@ -133,7 +133,7 @@ jobs:
 
           # Python build settings
           CIBW_BEFORE_BUILD: |
-            python -m pip install pybind11 cmake~=3.27.0 build
+            python -m pip install pybind11 cmake~=3.27.0 build scipy-openblas32
```

Comment:
I suppose `scipy-openblas32` is an installation dependency and we don't need to install this package before building the wheels. :thinking: